package.json

```diff
@@ -12,7 +12,8 @@
   "scripts": {
     "mocha": "mocha --exit",
     "xo": "xo",
-    "test": "npm run xo && npm run mocha",
+    "tsc": "tsc --esModuleInterop --noEmit test/index.ts",
+    "test": "npm run xo && npm run mocha && npm run tsc",
```

Comment:
Please move the tsc script to a separate script and call that in `test` like we do we the other two. This is so that one can run all scripts individually if they want to.

Reply:
Ummm... `npm run tsc` is just an alias for a specific but small `tsc` configuration, `tsc --esModuleInterop --noEmit test/index.ts`. It's like `npm run xo` -> `xo` and `npm run mocha` -> `mocha --exit`.

Or am I missing the point here?

Reply:
I think this is fine. Unless there's any strong objection, I'm ok to merge this. @XhmikosR ?

Reply:
Like I said, it's pretty handy to be able to run the scripts directly and that's what we are doing all this time.

Reply:
Actually, NVM, it looks good as it is 👍